hier_config/platforms/functions.py

```diff
@@ -24,12 +41,16 @@ def convert_to_set_commands(config_raw: str) -> str:
         config_raw (str): Configuration string
 
     """
-    lines = config_raw.split("\n")
+    lines: list[str] = config_raw.split(sep="\n")
+
+    # List of paths to the current command
     path: list[str] = []
+
+    # The list of actual configuration commands
     set_commands: list[str] = []
 
     for line in lines:
-        stripped_line = line.strip()
+        stripped_line: str = line.strip()
```

Comment:
I typically don't gratuitously add type annotations to variables unless there's some ambiguity pointed out by pyright/mypy --strict. Is there an argument for this practice?